.pre-commit-config.yaml

```diff
@@ -7,8 +9,10 @@ repos:
     - id: check-ast
     - id: check-case-conflict
     - id: check-merge-conflict
-    - id: end-of-file-fixer
     - id: trailing-whitespace
+      types_or: [python, c, c++, batch, markdown]
+    - id: end-of-file-fixer
```

Comment:
Ive seen cases where applying trim trailing whitespace will cause end of file to fail again, so that's why I swapped them.

Reply:
these don't appear swapped?

Reply:
`trailing-whitespace` is definitely above `end-of-file-fixer` now. It was the other way around before this pr.